src/productList.js

```diff
@@ -0,0 +1,29 @@
+import createRouter from "./router.js";
+import data from "../__mocks__/fileMock.js";
+
+const router = createRouter();
+
+export default function productList() {
+  const $productList = document.createElement("ul");
```

Comment:
```suggestion
  const productList = document.createElement("ul");
```
함수 앞에 $를 붙이신 이유가 있을까요?